app/apps/addresses/models.py

```diff
@@ -64,34 +69,40 @@ def __str__(self):
             )
         return self.bag_id
 
-    def get(bag_id):
+    def get_or_create(bag_id):
         return Address.objects.get_or_create(bag_id=bag_id)[0]
 
-    def save(self, *args, **kwargs):
+    def get_bag_address_data(self):
+        # When moving the import to the beginning of the file, a Django error follows:
+        # ImproperlyConfigured: AUTH_USER_MODEL refers to model 'users.User' that has not been installed.
         from utils.exceptions import DistrictNotFoundError
 
```

Comment:
Misschien kun je hier een vervolg ticket van maken want het is echt niet zoals het hoort, maar misschien ook lastig om snel op te lossen.
Als het over AUTH_USER_MODEL klaagt, kun je misschien kijken naar de lazy Django functie `get_user_model` in die file `utils.exceptions`